tuxedo-core/src/types.rs

```diff
@@ -47,6 +47,20 @@ pub struct Transaction<V: TypeInfo, C: TypeInfo> {
     pub checker: C,
 }
 
+impl<V: Clone, C: Clone> Transaction<V, C> {
```

Comment:
Not sure if this is better abstracted over a trait but since you cant use `From` here this is not a bad solution